.devcontainer/devcontainer.json

```diff
@@ -18,10 +18,19 @@
 		"stackbuild.bazel-stack-vscode-cc",
 		"augustocdias.tasks-shell-input",
 		"ryuta46.multi-command",
+		"zxh404.vscode-proto3",
+		"ms-python.python",
+		"ms-python.vscode-pylance",
+		"njpwerner.autodocstring",
```

Comment:
I think it would be useful to extract the changes to `devcontainer.json` into a very small PR that will already lift Python support from "nonexistent" to "rudimentary". That part could probably already be merged.

Reply:
Sounds good. I will prepare a devcontainer.json PR.